test/integration/activate_int_test.go

```diff
@@ -16,28 +16,33 @@ import (
 	"github.com/ActiveState/cli/internal/fileutils"
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
 )
 
 type ActivateIntegrationTestSuite struct {
-	suite.Suite
+	tagsuite.Suite
 }
 
 func (suite *ActivateIntegrationTestSuite) TestActivatePython3() {
+	suite.OnlyRunForTags("python", "activate", "critical")
```

Comment:
Can we centralize these tag values somewhere as constants and just use the constants here. Then we can both ensure that the tags are consistent as well as have a central place to check for which tags are available.